src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs

```diff
@@ -58,7 +58,7 @@ public async Task<ActionResult> Index(string hashedAccountId, string reservation
             {
                 var unhashedAccountId = _hashingService.DecodeValue(hashedAccountId);
                 response.Data.AccountViewModel = await _portalClient.GetAccount(unhashedAccountId);
-                response.Data.ApprenticeshipAdded = response.Data.AccountViewModel?.Organisations?.FirstOrDefault().Cohorts?.FirstOrDefault() != null && response.Data.AccountViewModel?.Organisations?.FirstOrDefault().Cohorts?.FirstOrDefault().Apprenticeships?.Count > 0;
+                response.Data.ApprenticeshipAdded = response.Data.AccountViewModel?.Organisations?.FirstOrDefault() != null && response.Data.AccountViewModel?.Organisations?.FirstOrDefault().Cohorts?.FirstOrDefault() != null && response.Data.AccountViewModel?.Organisations?.FirstOrDefault().Cohorts?.FirstOrDefault().Apprenticeships?.Count > 0;
```

Comment:
This line is super ugly and unwieldy we should probably Tech-debt a better solution for instances where we have no org, no cohort etc

Reply:
Ugly? Beauty is in the eye of the beholder :)